pkg/extensions/config/sync/config.go

```diff
@@ -42,6 +42,7 @@ type Content struct {
 }
 
 type Tags struct {
-	Regex  *string
-	Semver *bool
+	Regex       *string
+	RejectRegex *string
```

Comment:
Would call this "ExcludeRegex"

Reply:
and fix symbols everywhere else accordingly.

Reply:
@rchincha done.